src/Apps/Artist/Routes/WorksForSale/ArtistWorksForSaleRoute.tsx

```diff
@@ -74,3 +78,49 @@ export const ArtistWorksForSaleRouteFragmentContainer = createFragmentContainer(
     `,
   }
 )
+
+export const ArtistWorksForSaleRouteQueryRenderer: FC<{
```

Comment:
Is this the sort of thing where for every lazily-loaded query renderer we need to create a wrapper component like this that defines the behavior? Or would it be possible to create a lazy-load grid that accepts a `query` and `variables prop where we pass in what we need for the grid render? As we'll be likely deferring things all over, it would be nice to make this a bit more reusable. 

(Can push this into a separate PR)